Experiences/Experiences/Extensions/UIViewController+InfoAlert.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIViewController+InfoAlert.swift
+//  Experiences
+//
+//  Created by Craig Swanson on 3/22/20.
+//  Copyright © 2020 craigswanson. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+    
```

Comment:
Always a good idea to create an alert controller in an extension of UIViewcontroller.